### PR TITLE
Docs/template

### PR DESCRIPTION
**Pull request by hrutik7**
just a templatete for the docs #1

### CODE REVIEW

File: server.js
Comment (DonKoko) [inline server.js:1]: Not sure why we are doing this. We should always be using ts over js.

File: remix.init/index.js
Comment (DonKoko) [inline remix.init/index.js:1]: Can you give me some context about this file? First time I see it and not sure what's the purpose. Is it Vercel related?

File: scripts/cachePosts.ts
Comment (DonKoko) [inline scripts/cachePosts.ts:1]: Can you give me some context about this file? First time I see it and not sure what's the purpose. Is it Vercel related?

**Review by DonKoko — COMMENTED** [review]
Added some questions. Also I see you added the cache to the gitignore but it's still part of the PR. Should be removed.

**Comment by hrutik7** [comment]
closing this because I am facing some problems in local env after solving some merge conflicts

**Pull request closed without merging**